docs/documentation/server_admin/topics/users/con-aia.adoc

```diff
@@ -39,18 +39,35 @@ by checking the claims like `acr` in the tokens.
+In case the policy is not configured, it is also possible to configure it on the required action itself in the <<proc-setting-default-required-actions_{context}, Required actions tab>>
+when configuring the particular required action. If not configured anywhere from those places, it defaults to five minutes.
 
 * If you want to use a shorter re-authentication, you can still use a parameter query parameter such as  `max_age` with the specified shorter value or eventually `prompt=login`, which will always require user to
 actively re-authenticate as described in the OIDC specification. Note that using `max_age` for a longer value than the default five minutes (or the one prescribed by password policy) is not supported.
 The `max_age` can be currently used only to make the value shorter than the default five minutes.
 
+* In case that <<_step-up-flow,Step-up authentication>> is enabled and the action is either adding the new credential or deleting the existing credential, then it is required to authenticate with the level corresponding
+to the given credential in case that user already has the credential of the particular level. For example if there are `otp` and `webauthn` configured in authentication flow as 2nd-factor authenticators
+(both in the authentication flow at level 2) and user already has some existing 2nd-factor credential (`otp` or `webauthn` in this case), then for adding another 2nd-level credential, he is required to authenticate with some existing 2nd-factor.
+Similarly for delete existing 2nd-factor credential (`otp` or `webauthn` in this case), it is needed to authenticate with some existing 2nd-factor level credential. This is for security reasons.
+
+[id="con-aia-parameterized_{context}"]
+== Parameterized AIA
+
+Some AIA can require the parameter to be sent together with the action name. For instance action `Delete Credential` can be triggered only by AIA and it requires parameter to be sent together with the name
+of the action, which points to the ID of the removed credential. So for example in the URL it can be used like `kc_action=delete_credential:ce1008ac-f811-427f-825a-c0b878d1c24b` where in this case the
+part after the colon character (in this example the value `ce1008ac-f811-427f-825a-c0b878d1c24b`) contain the ID of the credential of the particular user, which is supposed to be deleted. The `Delete Credential`
+displays the confirmation screen where user can confirm if he agrees with deleting the particular credential.
+
+NOTE: The <<_account-service,{project_name} Account console>> typically uses `Delete Credential` action when deleting some 2nd-factor credential, so you can take a look at the account console for the inspiration if you want
+to use this action directly from your own applications for some reason. But whenever possible, it is recommended to rely on the account console when possible instead of managing credentials from your own applications.
+
```

Comment:
```suggestion
NOTE: The <<_account-service,{project_name} Account Console>> typically uses the `Delete Credential` action when deleting a 2nd-factor credential.  You can check the Account Console for examples if you want
to use this action directly from your own applications. However, relying on the Account Console is best instead of managing credentials from your own applications.

```